packages/api/tests/endpoints/test-executions.js

```diff
@@ -1660,3 +1678,55 @@ test.serial('PUT /executions with non-existing parentArn still updates the execu
   t.is(fetchedPgRecord.arn, fetchedDynamoRecord.arn);
   t.falsy(fetchedPgRecord.parent_cumulus_id);
 });
+
+test.serial('PUT /executions publishes message to SNS topic', async (t) => {
+  const {
+    executionPgModel,
+    knex,
+    QueueUrl,
+  } = t.context;
+
+  const newExecution = fakeExecutionFactoryV2({
```

Comment:
I'd consider using the fakeExecutionRecordFactory function

Reply:
I don't think that will work. The POST /executions endpoint calls `writeExecutionRecordFromApi`:

https://github.com/nasa/cumulus/blob/feature/rds-phase-2/packages/api/endpoints/executions.js#L61

And that function expects the incoming record to be a Dynamo record, which is then translated to the PostgreSQL record:

https://github.com/nasa/cumulus/blob/feature/rds-phase-2/packages/api/lib/writeRecords/write-execution.js#L216-L228

So we can't use `fakeExecutionRecordFactory` because it would be sending a PostgreSQL-style record to the endpoint, not a Dynamo/API style record as it expects